spec/slack_notifier_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "spec_helper"
+require "./slack_notifier"
+
+describe SlackNotifier do
+  before :each do
+    stub_request(:any, "https://hooks.slack.com")
+  end
+  let(:notifier) { SlackNotifier.new("https://hooks.slack.com") }
+
+  describe ".new_issue_not_published" do
+     it "sends correct payload to slack" do
+       notifier.new_issue_not_published
+       expect(WebMock).to have_requested(:post, "https://hooks.slack.com").
+                            with(body: '{"text":"*Fíha, dnes na ÚVO nevyšlo nové vydanie vestníka?*","channel":"#general","username":"uvobot","icon_emoji":":mag_right:"}')
+     end
+  end
+
+  describe ".matching_announcements" do
+    let(:announcements) {[
+      { link: { text: "text 1", href: "href 1" }, customer: "customer 1", description: "desc 1" }
+    ]}
+
+    context "announcements found" do
+      it "sends correct payloads to slack" do
+        notifier.matching_announcements("Found 1 record", announcements)
+
+        expect(WebMock).to have_requested(:post, "https://hooks.slack.com").
+                             with(body: '{"text":"Našiel som niečo nové na ÚVO! (Found 1 record)","channel":"#general","username":"uvobot","icon_emoji":":mag_right:"}')
```

Comment:
Line is too long. [168/80]<br>Use 2 (not 21) spaces for indenting an expression spanning multiple lines.